src/processor/stackwalker.cc

```diff
@@ -321,6 +327,30 @@ Stackwalker* Stackwalker::StackwalkerForCPU(
                                              frame_symbolizer);
       break;
     }
+
+    case MD_CONTEXT_RISCV:
+    {
+      const MDRawContextRISCV* context_riscv = context->GetContextRISCV();
+      // memory = GetActualStackMemory(context_riscv->__gregs[MD_CONTEXT_RISCV_REG_SP],
+      //                               memory, memory_list);
+      cpu_stackwalker = new StackwalkerRISCV(system_info,
+                                             context_riscv,
+                                             memory, modules,
+                                             frame_symbolizer);
+      break;
+    }
+
+    case MD_CONTEXT_RISCV64:
+    {
+      const MDRawContextRISCV64* context_riscv64 = context->GetContextRISCV64();
+      // memory = GetActualStackMemory(context_riscv64->__gregs[MD_CONTEXT_RISCV_REG_SP],
+      //                               memory, memory_list);
+      cpu_stackwalker = new StackwalkerRISCV64(system_info,
+                                               context_riscv64,
+                                               memory, modules,
+                                               frame_symbolizer);
+      break;
+    }
```

Comment:
This is the only conflict (out of two) I could not resolve: haven't figured out where the stack-pointer register is.


https://github.com/getsentry/breakpad/pull/1